### PR TITLE
[FIX] Languages are written as codes

### LINKED ISSUE #20330: Some languages are written in codes
### Steps to reproduce:

1. Go to Account Settings > Preferences
2. Click on Languages in Localisation

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.0 - develop
- Operating System: Ubuntu
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: Chrome
- Operating System: Windows

### PR DESCRIPTION
**Pull request by im-adithya**
## Proposed changes (including videos or screenshots)

I changed the names of the files in i18n folder

## Issue(s)

This PR closes #20330 
## Earlier
Screenshot (19)
Screenshot (25)
Screenshot (20)

## Now
Screenshot (23)
Screenshot (24)
Screenshot (22)

### CODE REVIEW

**Comment by im-adithya** [comment]
Can I change other codes in this PR itself? Some languages are not in this scroll bar so I left some of them.

**Comment by gabriellsh** [comment]
Yep, I think it's good to have everything in a single PR

**Comment by im-adithya** [comment]
Sure @gabriellsh I'll include them too!

**Comment by im-adithya** [comment]
Screenshot (45)
Screenshot (46)
Screenshot (47)
Screenshot (48)
Screenshot (49)
Screenshot (50)
Screenshot (51)
Screenshot (52)
Screenshot (53)
Screenshot (54)
Screenshot (55)
Screenshot (56)
Screenshot (57)

@gabriellsh I think some are rendered properly only when the country code is specified.
Now everything is fine as shown in the screenshots!

File: packages/rocketchat-i18n/i18n/bas-CM.i18n.json [hunk 1]
```diff
@@ -1 +1 @@
-{ }
\ No newline at end of file
+{ }
```
Comment (KevLehman) [inline packages/rocketchat-i18n/i18n/bas-CM.i18n.json:1]: 🤔

**Pull request closed without merging**